cs/cli/vw_example.cpp

```diff
@@ -139,7 +139,7 @@ namespace VW
 
     System::String^ FormatFeature(vw* vw, feature_value& f1, feature_index& i1)
     {
-        uint64_t masked_weight_index1 = i1 & vw->reg.weight_mask;
+		uint64_t masked_weight_index1 = i1 & vw->wv.mask();
```

Comment:
Since code like this is all over the place, have you considered adding an inline "index" function to your weigh_vector class?